.github/workflows/main.yml

```
@@ -5,11 +5,22 @@ on: [push, pull_request]
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - "lint"
+          - "test-centos:6"
+          - "test-centos:7"
+          - "test3-centos:stream8"
+          - "test3-centos:stream9"
     steps:
       - uses: actions/checkout@v2.4.0
```

Oh, and unrelated but isn't this good enough:
```suggestion
      - uses: actions/checkout@v2
```